### ISSUE TITLE
Help! Can't revert HighDPI scaling factor. 

### ISSUE THREAD

**Issue by Diego-MX** [spyder-ide/spyder#5396]
## Problem 
Was experimenting with HighDPI scaling factor and now I can't revert it.  
I think the problem is that the scaling factor doesn't render the window properly, from which I cannot see the properties that would take my old settings back.  

**What steps will reproduce the problem?**
1.  I first clicked the property of Enable HighDPI (this is the name I remember, but I can't see it now). 
  Then I changed the scaling factor to 0.8.  I don't know why I tried something smaller than 1 first.  My mistake. 
2.  Close and open Spyder. 

**What is the expected output? What do you see instead?**
I wanted to see how it affected the window.  I expected larger icons and text, and obviously chose the wrong way to go with 0.8.
I can't copy the dependencies because there is no Copy button, but I found the way to bring it about and do it manually.

**Please provide any additional information below**

## Versions and main components

* Spyder Version: 3.2.1
* Python Version: 3.6.1 Anaconda  (got it from terminal)
* Qt Version:  cant find it
* PyQt Version: cant find it
* Operating system:  Ubuntu 16.04

## Dependencies

Please go to the menu entry `Help > Optional Dependencies` (or
`Help > Dependencies`), press the button `Copy to clipboard`
and paste the contents below:
IPython 5.3cython 0.25.2jedi 0.10.2
nbconvert 5.1.1
numpy .12.1
pandas 0.20.1
psutil 5.2.2
pycodestyl 2.3.1
pyflakes 1.5.
pygments 2.2
pylin 1.6.4
qtconsole 4.3
rope 0.0.4-1
sphinx 1.5.6
sympy 1.0

**Comment by rlaverde** [comment]
Running `spyder --reset` will reset to default preferences

**Comment by Diego-MX** [comment]
Great.  Thanks.

**Comment by rlaverde** [comment]
Although I think we should add an easy way to revert this, because if it's misconfigured the only way to revert it is reseting all preferences 😕

**Comment by Diego-MX** [comment]
This did it for me; but yes a small inconvenience was that other preferences were reset. 

After seeing the 1.1 DPI scaling factor work when set properly, I would expect a factor of 0.8 to make it a little bit smaller, but from before it broke the application.
My two cents contribution would be to see what happens with smaller factors, or to set a restriction into which factors to use. 

Thanks.

**Comment by ccordoba12** [comment]
@rlaverde, a menu entry in `Tools` to reset DPI config would be good to have. Do you want to work on it?

**Comment by rlaverde** [comment]
>a menu entry in Tools to reset DPI config would be good to have. Do you want to work on it?

I'm not sure if that could work, if you set a wrong value is really hard to interact with the UI

**Comment by ccordoba12** [comment]
I see. The we would need to add a new command line option for this.

**Comment by Diego-MX** [comment]
That's true.  The problem was not the lack of menu item, but being unable to interact with the UI.  

In the image above, not only were the elements in the preferences' boxes missing, but their actual position didn't correspond to where the mouse was hovering over.  The same happened to the menu items File, Edit, Search, etc...

**Comment by ccordoba12** [comment]
@rlaverde, do you want to work on this one?

**Comment by rlaverde** [comment]
yes, adding a command line option? something like `spyder --reset-hdpi`?

**Comment by ccordoba12** [comment]
Yep, that would be the idea. Perhaps the option should be

    --reset-hdpi-settings
